src/UnitTest/Geometry/TriangleMesh.cpp

```diff
@@ -1437,92 +1437,92 @@ TEST(TriangleMesh, DeformAsRigidAsPossible) {
 
 TEST(TriangleMesh, SelectByIndex) {
```

Comment:
Since we're here, would it make sense to change this random test to something that we know the ground-truth?
E.g. a mesh with several vertices and triangles, where we call `SelectByIndex` to get a subset of vertices.

Reply:
Yes, but I think this does not only apply to this test/PR. Better address this in another PR?

Reply:
sure